.gitignore

```diff
@@ -69,3 +69,13 @@ target/
 .xml
 .shx
 .cpg
+
+#IDE folder
+.vscode
```

Comment:
keep this line but remove all the others

Reply:
I donot get it. If so (all the other lines are removed), the IPython notebook file will be committed, which is not my intention.

Reply:
If I understand correctly, you committed a bunch of files and pushed those changes. When you got to commit the changes you should only add/commit the specific files that you want to add to the repo or update in the repo.

So say I add some new files locally called `test.py` and `test2.py`. And I only want to commit `test.py` then I would just do `git add test.py` and then `git commit -m "add test.py"`  and then git push. Then only `test.py` would be committed and pushed the server. And I don't have to update the `.gitignore` file. 

I'm guessing maybe you committed an entire directory and that included all of the files that were add/changed.

Reply:
Thanks, now I understand. I figure out how to use git to revert files and remove cache. Please have a look at my latest commit. Let me know if there is anything that needs to be improved.